pkg/dns/dns.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package dns
 
 import (
+	"context"
 	"github.com/Kuadrant/multicluster-gateway-controller/pkg/apis/v1alpha1"
```

Comment:
import grouping. Keep the stdlib imports separate at the top.

Reply:
sorry i thought the make fmt sorted all that let me fix it